agogosml/agogosml/common/kafka_streaming_client.py

```diff
@@ -53,9 +50,26 @@ def create_kafka_config(user_config: dict) -> dict:
         config = {
             "bootstrap.servers": user_config.get("KAFKA_ADDRESS"),
             "enable.auto.commit": False,
-            "auto.offset.reset": "earliest"
+            "auto.offset.reset": "earliest",
+            # 'request.timeout.ms': 60000,
+            # 'session.timeout.ms': 60000,
+            # 'default.topic.config': {'auto.offset.reset': 'smallest'},
         }
 
+        if user_config.get('EVENTHUB_KAFKA_CONNECTION_STRING'):
+            ssl_location = user_config.get('SSL_CERT_LOCATION')
+            if user_config.get('SSL_CERT_LOCATION') is None:
+                ssl_location = '/etc/ssl/ca-certificates.crt'
```

Comment:
You can use `dict.get(key, default)` to collapse this block:

```py
ssl_location = user_config.get('SSL_CERT_LOCATION', '/etc/ssl/ca-certificates.crt')
```

Or even better, to handle defaults if the key is set to None explicitly:

```py
ssl_location = user_config.get('SSL_CERT_LOCATION') or '/etc/ssl/ca-certificates.crt'

```